master/getting-started/kubernetes/installation/hosted/canal/canal-etcd.yaml

```diff
@@ -1,515 +1,4 @@
-
-# This ConfigMap can be used to configure a self-hosted Canal installation.
-kind: ConfigMap
-apiVersion: v1
-metadata:
-  name: canal-config
-  namespace: kube-system
-data:
-  # Configure this with the location of your etcd cluster.
-  etcd_endpoints: "https://127.0.0.1:2379"
-
-  # The interface used by canal for host <-> host communication.
-  # If left blank, then the interface is chosing using the node's
-  # default route.
-  canal_iface: ""
-
-  # Whether or not to masquerade traffic to destinations not within
-  # the pod network.
-  masquerade: "true"
-
-  # The CNI network configuration to install on each node.  The special
-  # values in this config will be automatically populated.
-  cni_network_config: |-
-    {
-        "name": "canal",
-        "cniVersion": "0.3.0",
-        "plugins": [
-            {
-                "type": "flannel",
-                "delegate": {
-                    "type": "calico",
-                    "etcd_endpoints": "__ETCD_ENDPOINTS__",
-                    "etcd_key_file": "__ETCD_KEY_FILE__",
-                    "etcd_cert_file": "__ETCD_CERT_FILE__",
-                    "etcd_ca_cert_file": "__ETCD_CA_CERT_FILE__",
-                    "log_level": "info",
-                    "policy": {
-                        "type": "k8s",
-                        "k8s_api_root": "https://__KUBERNETES_SERVICE_HOST__:__KUBERNETES_SERVICE_PORT__",
-                        "k8s_auth_token": "__SERVICEACCOUNT_TOKEN__"
-                    },
-                    "kubernetes": {
-                        "kubeconfig": "/etc/cni/net.d/__KUBECONFIG_FILENAME__"
-                    }
-                }
-            },
-            {
-                "type": "portmap",
-                "capabilities": {"portMappings": true},
-                "snat": true
-            }
-        ]
-    }
-
-  # If you're using TLS enabled etcd uncomment the following.
-  # You must also populate the Secret below with these files.
-  etcd_ca: "" # "/calico-secrets/etcd-ca"
-  etcd_cert: "" # "/calico-secrets/etcd-cert"
-  etcd_key: "" # "/calico-secrets/etcd-key"
-
----
-# The following contains k8s Secrets for use with a TLS enabled etcd cluster.
-# For information on populating Secrets, see http://kubernetes.io/docs/user-guide/secrets/
-apiVersion: v1
-kind: Secret
-type: Opaque
-metadata:
-  name: calico-etcd-secrets
-  namespace: kube-system
-data:
-  # Populate the following files with etcd TLS configuration if desired, but leave blank if
-  # not using TLS for etcd.
-  # This self-hosted install expects three files with the following names.  The values
-  # should be base64 encoded strings of the entire contents of each file.
-  # etcd-key: ""
-  # etcd-cert: ""
-  # etcd-ca: ""
-
----
-
-# This manifest installs the per-node agents, as well
-# as the CNI plugins and network config on
-# each master and worker node in a Kubernetes cluster.
-kind: DaemonSet
-apiVersion: extensions/v1beta1
-metadata:
-  name: canal-node
```

Comment:
looks like the name of the daemonset changes to `canal` (instead of `canal-node`) in the new manifests. This would break upgrades for existing users applying our manifests directly...